### ISSUE TITLE
Usage of default buffer for table - CAN-FIND function

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#456]
````
define temp-table tt field ii as int .

create tt. tt.ii = 1.
create tt. tt.ii = 2.

message can-find( tt where tt.ii = 1 ) tt.ii view-as alert-box.
````
The CAN-FIND function does not use buffers.

Note that the example above illustrates that can-find does not use a buffer, to trigger the default buffer rule a database buffer must be used:

````
procedure one:

   message can-find( first customer ).

end procedure.
````

**Comment by gquerret** [comment]
Right, it seems the rule shouldn't be triggered within `CAN-FIND` statements

**Comment by gquerret** [comment]
Fixed in upcoming 2.1.5
